vss-extension.json

```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "infracost-tasks",
   "name": "Infracost Tasks",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
Something looks odd in https://github.com/infracost/infracost-azure-devops/tags, is v0.1.4 meant to be v1.0.4?

Reply:
yeah must be